### DISCUSSION TITLE
Premium templates issues

### DISCUSSION THREAD

**Discussion by arustamyansv** [primefaces/community#1879] [PrimeVue Templates] [answered]
Dear Primevue Team,

I have the extended license for the Atlantis template.

When I'm importing Figma file to my project it is extremely scarce compared to what I can see in Atlantis examples online. 
As if it's not exported fully. I can see icons, typography, colors, some of the pages, and the dashboard but it lacks separate components. 
Could it be it's not exported properly? The version I have for download in the interface is 1.0.

Also, I originally bought this scheme to pair with v4 of Primevue since there is a link to purchase from v4 Primevue site (which in my opinion is very misleading). I saw a person from your team answer in one of the threads that since Primevue is still in beta you didn't migrate templates... Do you have any estimate on when it would be available?

Thanks in advance!

**Comment by davidhernandeze** [comment]
I agree, having those templates under v4 documentation is confusing.

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

We are planning to complete all premium templates by early September. We will adhere to the roadmap for template updates.

Thanks for your understanding!
